### ISSUE TITLE
Screen reader doesn't read full displayed text on toolbar buttons.

### ISSUE THREAD

**Issue by sculpt0r** [ckeditor/ckeditor4#4441]
## Type of report

Bug

## Provide detailed reproduction steps (if any)

1. Run the `JAWS` screen reader.
2. Open CKEditor4 in [High Contrast Mode](https://ckeditor.com/docs/ckeditor4/latest/guide/skin_sdk_hc.html).
3. [Nawigate with keyboard ](https://ckeditor.com/docs/ckeditor4/latest/guide/dev_a11y.html#keyboard-navigation)to toolbar buttons.
4. With `tab` move selection to button with name and shortcut description on it, e.g. `ItalicKeyboard shortcut Ctrl+I`.
5. Listen to screen reader voice.

### Expected result

It should be read full visible button text with a pause before `Keyboard` word. Ended with `button` word.

So for an example from point 4. - it should be: `Italic Keyboard shortcut control plus I button`

### Actual result

The screen reader reads: `Italic button`.

## Other details
It was discovered along with solving #4422 
The button label is divided into two `span` elements:
- first contains the text: `Italic`
- second contains text: `Keyboard shortcut Ctrl+I`

Main parent (`a`) contains `aria-labelledby` attribute pointing at first `span` label. Also, contains `aria-describedby` attribute pointing at the second `span` label. 

As `aria-describedby` is an additional description - seems that the screen reader does it right. To force all visible text to be read, maybe we should redesign toolbar buttons elements? 

Screen reader also has its own settings about reading elements - so overall it might change expected behavior.

* Browser: Firefox (newest)
* OS: Windows 10
* CKEditor version: 4.15.1
* Installed CKEditor plugins: -
* Screen Reader: JAWS

**Comment by sculpt0r** [comment]
Similar (?): https://github.com/ckeditor/ckeditor4/issues/3734
